mibidata/tests/test_segmentation.py

```diff
@@ -175,6 +175,23 @@ def test_expand_objects(self):
         assert_array_equal(segmentation.expand_objects(labels, 1), expected_1)
         assert_array_equal(segmentation.expand_objects(labels, 2), expected_2)
 
+    def test_adjacency_matrix(self):
+        cell_labels = np.array([
```

Comment:
According to your definition, all of the points in this label image are boundaries. Maybe it would be nice if you have some non-boundary points as well.

Reply:
ok, ok, I'll do that